main.tf

```diff
@@ -115,8 +115,8 @@ resource "google_container_cluster" "primary" {
   */
 
   master_auth {
-    username = "admin"
-    password = random_string.password.result
+    username = ""
+    password = ""
```

Comment:
This looks like an unrelated change, do we need this change?

Reply:
i can remove this completely , basic auth is no more supported

Reply:
basic auth is dropped from 1.19

Reply:
aah ok, I will defer it to @danielhoherd

Reply:
i think we should leave these fields in here because we can use this module with clusters that are earlier than 1.19.